README.md

```diff
@@ -22,7 +22,15 @@ git clone git@github.com:DD2480-Project-group-25/Continuous-Integration.git
 
 ### Database with logged history
 
-An HTTP post request is used to log history from the CI service runs. The API for posting is available at ```http://localhost/8080/api/```. The CI service will post to this API if you run ```python3 manage.py runserver 8080``` standing in ```./logDb```.
+An HTTP post request is used to log history from the CI service runs. The API for posting is available at ```http://log-srv:8080/api/```. The CI service will post to this API if you run ```python3 manage.py runserver 8080``` standing in ```./logDb```.
```

Comment:
This is not entirely true anymore, as the ci-server will post to http://log-srv:8080/api/ which will bind to the log server trough docker.